inc/classes/class-genesis-featured-widget-amplified.php

```diff
@@ -1220,27 +1198,27 @@ function gfwa_do_post_meta( $instance ) {
 }
 
 /**
- * Returns "display: none;" if option and value match, or of they don't match with $standard is set to false
+ * Returns "display: none;" if option and value match, or if they don't match with $standard set to false.
  *
  * @author Nick Croft
  * @since 0.8
  * @version 0.8
  * @param array   $instance Values set in widget $instance.
  * @param mixed   $option instance option to test.
  * @param mixed   $value value to test against.
- * @param boolean $standard echo standard return false for oposite.
+ * @param boolean $standard echo standard return false for opposite.
  * @return string
  */
 function gfwa_get_display_option( $instance, $option = '', $value = '', $standard = true ) {
 	$display = '';
 	if ( is_array( $option ) ) {
 		foreach ( $option as $key ) {
-			if ( in_array( $instance[ $key ], $value ) ) { // WPCS: loose comparison ok.
+			if ( in_array( $instance[ $key ], $value, false ) ) {
 				$display = 'display: none;';
 			}
 		}
 	} elseif ( is_array( $value ) ) {
-		if ( in_array( $instance[ $option ], $value ) ) { // WPCS: loose comparison ok.
```

Comment:
Needs loose here too

Reply:
See above explanation.

For kicks, run PHPCS with the `<exclude name="WordPress.PHP.StrictInArray.FoundNonStrictFalse"/>` commented out, and the warning will appear. Try adding this `// WPCS....` comment back in, and you'll see it has no effect.